src/librustc_codegen_llvm/context.rs

```diff
@@ -99,6 +99,16 @@ fn to_llvm_tls_model(tls_model: TlsModel) -> llvm::ThreadLocalMode {
     }
 }
 
+fn to_llvm_code_model(code_model: CodeModel) -> llvm::CodeModel {
```

Comment:
`fn to_llvm_code_model` already exists in `src\librustc_codegen_llvm\back\write.rs`.